### DISCUSSION TITLE
I need help to create a Filter

### DISCUSSION THREAD

**Discussion by Ghustavitto** [Chatterino/chatterino2#5670] [Q&A] [answered]
I would like to know how to leave an Automod message like the one in the photo, in highlight.
![image](https://github.com/user-attachments/assets/0440dff6-7cf4-40c7-ae57-90347af56a18)
![image](https://github.com/user-attachments/assets/cf3bd7e5-1a7d-4158-b08d-ffd8a3025963)

**Comment by Felanbird** [comment, marked as answer]
We don't have highlights for system messages, but since a filter would allow for a split with only certain messages, it would work.

for any system message
```
flags.system_message
```

for only timeouts
```
flags.system_message && message.content contains "timed"
```

Don't forget to select `set filters` in the `...` menu
![image](https://github.com/user-attachments/assets/fbd1a55e-52b9-404d-b8ef-c0165972dc33)

**Reply by Felanbird** [reply]
I also use this filter in a channel so normal messages will appear when the channel is offline, but it only shows the mod actions when they are live.

```
(channel.live && (flags.system_message && !flags.sub_message)) || (!channel.live)
```

**Reply by Ghustavitto** [reply]
Thank you so much!
